tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -614,18 +621,25 @@ public ImmutableTracerSettings Build()
             return headerTags;
         }
 
-        // internal for testing
-        internal static IEnumerable<string> TrimSplitString(string textValues, char separator)
+        internal static string[] TrimSplitString(string textValues, char[] separators)
```

Comment:
In .NET Framework, each call with a single `char` uses `String.Split(params char[], ...)`, which allocates a new array. Instead, I'm creating the `char[]` explicitly and re-using it in several calls to `TrimSplitString()`.

Admittedly, it's a tiny optimization since there are only 3 such calls during initialization.

Reply:
:TIL: